Sprint-3/quote-generator/style.css

```diff
@@ -1 +1,56 @@
 /** Write your CSS in here **/
+body {
+  font-family: Arial, sans-serif;
+  background-color: #f5f5f5;
+  color: #333;
+  text-align: center;
+  padding: 20px;
+}
+
+.container {
+  max-width: 600px;
+  margin: 50px auto;
+  padding: 20px;
+  background-color: white;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  border-radius: 10px;
+}
+
+h1 {
+  font-size: 2rem;
+  color: #007bff;
+}
+
+.quote-box {
+  margin: 20px 0;
+  padding: 20px;
+  border-left: 4px solid #007bff;
+  background-color: #f9f9f9;
+  border-radius: 5px;
+}
+
+#quote {
+  font-size: 1.5rem;
+  font-style: italic;
+}
+
+#author {
+  font-size: 1.2rem;
+  color: #555;
+  margin-top: 10px;
+}
+
+button {
+  background-color: #007bff;
+  color: white;
+  border: none;
+  padding: 10px 20px;
+  font-size: 1rem;
+  cursor: pointer;
+  border-radius: 5px;
+  transition: background-color 0.3s ease;
+}
+
+button:hover {
+  background-color: #0056b3;
+}
```

Comment:
It is nice to see that you gave importance on styling. You can also use nesting operator while using hover 
https://developer.mozilla.org/en-US/docs/Web/CSS/Nesting_selector you can check this page for more information